tasks.php

```diff
@@ -1602,32 +1602,36 @@ function TaskComments($tid)
 
 function NotificationMail($tid, $message, $new_task = false)
 {
-    global $code_url, $tasks_url, $pguser;
+    global $code_url, $tasks_url, $pguser, $date_str, $time_of_day_str;
+
+    $result = mysqli_query(DPDatabase::get_connection(), "SELECT task_summary, task_details FROM tasks WHERE task_id = $tid LIMIT 1");
+    if(!$result)
+        return;
+    $row = mysqli_fetch_assoc($result);
+    $task_summary = $row['task_summary'];
+
+    $message = $message . "\nYou can see task #$tid by visiting $tasks_url?action=show&task_id=$tid\n\n--\n
```

Comment:
`action=show` is the default so it isn't needed in the url, eg: https://www.pgdp.org/c/tasks.php?task_id=12

Reply:
Also, while the former method down near line 164 was more "wordy" with each message line being on its own line in the PHP code, it made it much easier to read.

Reply:
Being an old and low-level programmer it's hard to get out of the habit of making the code more efficient (like making two strings into one instead of letting the computer do it). I'll put it back like it was.